llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -1261,6 +1261,9 @@ const EnumEntry<unsigned> ElfMachineType[] = {
   // EraVM local begin
   ENUM_ENT(EM_ERAVM,         "EraVM"),
   // EraVM local end
+  // EVM local begin
+  ENUM_ENT(EM_EVM,           "Ethereum Virtual machine"),
```

Comment:
Please make consistent with the `EraVM` above.

Reply:
Done.